src/openforms/authentication/forms.py

```diff
@@ -45,10 +45,9 @@ class RegistratorSubjectInfoForm(forms.Form):
     )
 
     def clean(self):
-        cleaned_data = super().clean()
-        bsn = cleaned_data.get("bsn")
-        kvk = cleaned_data.get("kvk")
-        skip_subject = cleaned_data.get("skip_subject")
+        bsn = self.cleaned_data.get("bsn")
```

Comment:
Why did we remove the `super()` call?Isn't it more safe concerning the values?I had in mind that we first call the `super().clean()` and then do what we need with these values.

Reply:
`super().clean()` can return `None`, there's no guarantee that it will actually return `self.cleaned_data`. This is something that changed/got clarified around Django 1.11-2.2 if I remember correctly.

I do need to add the `super` call back though